src/config/config.go

```diff
@@ -96,4 +98,7 @@ func ProcessArgs() {
 			printHelpAndExit()
 		}
 	}
+	if ret.InfraEnvID == "" {
```

Comment:
In which case will it happen?

Reply:
we cannot add https://github.com/openshift/assisted-service/pull/2326, until assisted-installer supports its. So, until that PR is pushed, we won't get the infra-env id in command line, and we are setting it to cluster id

Reply:
when is the command line used?
I'm not familiar with that

Reply:
The command line @yevgeny-shnaidman refers to is the install-command that executes the assisted-installer on the node.

Will we have InfraEnvID when installing without kube-api e.g cloud.redhat.com?

Reply:
yes, install command is per host, and each host has its own infra-env id